products/ubuntu2204/profiles/cis_level2_workstation.profile

```diff
@@ -37,7 +37,8 @@ selections:
     - kernel_module_usb-storage_disabled
 
     #### 1.6.1.4 Ensure all AppArmor Profiles are enforcing (Automated)
-    #- ensure_apparmor_enforce
+    - var_apparmor_mode=enforce
```

Comment:
This variable is not used by `all_apparmor_profiles_enforced` rule. It is intentional to define it here?

Reply:
Thanks @dodys